guides/introduction.md

```diff
@@ -0,0 +1,177 @@
+# Introduction
```

Comment:
it could be cool to include a gif of your increment system with hot reloading here (im a fan of gifs as the first thing people see)